src/yadism/coefficient_functions/asy/partonic_channel.py

```diff
@@ -63,7 +61,7 @@ def sing(z, _args):
                 asnorm
                 * icl
                 * constants.CF
-                * ((1.0 + z**2) / (1.0 - z) * (l - 2.0 * np.log(1.0 - z) - 1.0))
+                * ((1.0 + z**2) / (1.0 - z) * (-2.0 * np.log(1.0 - z) - 1.0))
```

Comment:
No wait, I'm not sure: if there's a log, the log should stay, isn't it?

Can we have the original expression? It might be in some notebook in this repo, but maybe @felixhekhorn recalls better than me...

Reply:
Now I'm no longer sure any longer ... and in fact it is reintroduced in https://github.com/NNPDF/yadism/pull/195/commits/961357a8940a9af8ff05f34d86b830bb2bb79973 - however, I don't understand why ... I did it because of the game I was playing in [test_nc_asy.py](https://github.com/NNPDF/yadism/pull/195/commits/961357a8940a9af8ff05f34d86b830bb2bb79973#diff-20039f8f2fd50247add715384e577325eaf35a0cc88757e8b51edf6800b05086) in order to prove my previous statement - well it turned out with the log the difference is much better, so maybe I was just wrong and so I reintroduced it ...

On pure theory grounds I stand with what I said [earlier](https://github.com/NNPDF/yadism/pull/195#issuecomment-1580729465) ...

Reply:
> Can we have the original expression? It might be in some notebook in this repo, but maybe @felixhekhorn recalls better than me...

I believe we had never the full expression in MMa, but only parts (like fhat), or at least I cannot find it ... I believe we only went through Fortran (and translated from there)